tools/cloudflow-sbt-plugin/src/main/scala/cloudflow/sbt/CloudflowLocalRunnerPlugin.scala

```diff
@@ -313,6 +314,21 @@ object CloudflowLocalRunnerPlugin extends AutoPlugin {
     printAppLayout(resolveConnections(appDescriptor))
   }
 
+  def saveApplicationGraph: Def.Initialize[Task[File]] = Def.task {
+    implicit val logger = streams.value.log
+    val _appDescriptor = applicationDescriptor.value
+    val blueprintBaseDir = baseDirectory.value / "src" / "main" / "blueprint"
```

Comment:
Considering that this is not going to be a source file interpreted, but is a visualization for the users, it should go to a `target` folder.
You can even make that destination tunable with an additional `settingKey` if you have a specific use case for it.

Can you move it to `targetDirectory.value / <something>`?

Reply:
Done!